File: MPI/mpicheck/tests/functions/4.0/MPI_Pready_list.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    const int var_1[2];
    MPI_Request var_2;
    int ret;
    /* calls */
    ret = MPI_Pready_list(var_0, var_1, var_2);
    ret = PMPI_Pready_list(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Pready_list.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1(10)
       INTEGER var_2
       INTEGER var_3
        call mpi_pready_list(var_0, var_1, var_2, var_3)
       call pmpi_pready_list(var_0, var_1, var_2, var_3)
        end program main
    